exec/show.go

```diff
@@ -0,0 +1,60 @@
+package exec
+
+import (
+	"database/sql"
+	"strings"
+
+	"github.com/pkg/errors"
+	"go.knocknote.io/octillery/debug"
+	"go.knocknote.io/octillery/sqlparser"
+)
+
+// ShowQueryExecutor inherits QueryExecutorBase structure
+type ShowQueryExecutor struct {
+	*QueryExecutorBase
+}
+
+// NewShowQueryExecutor creates instance of ShowQueryExecutor
+func NewShowQueryExecutor(base *QueryExecutorBase) *ShowQueryExecutor {
+	return &ShowQueryExecutor{base}
+}
+
+// Query doesn't support in ShowQueryExecutor, returns always error.
+func (e *ShowQueryExecutor) Query() ([]*sql.Rows, error) {
+	return nil, errors.New("ShowQueryExecutor cannot invoke Query()")
+}
+
+// QueryRow doesn't support in ShowQueryExecutor, returns always error.
+func (e *ShowQueryExecutor) QueryRow() (*sql.Row, error) {
+	return nil, errors.New("ShowQueryExecutor cannot invoke QueryRow()")
+}
+
+// Exec executes `SHOW TABLE` DDL for shards
+func (e *ShowQueryExecutor) Exec() (sql.Result, error) {
+	debug.Printf("show table for shards")
```

Comment:
Not only is this method called on `show create table`, there are other possibilities (e.g. `show status` ). 
Therefore, this API should implement switching to `show create table` routine if `Table` parameter exists .

Reply:
Also, I think `SHOW CREATE TABLE` expect `QueryRow` API. Please confirm !

Reply:
Change to support `Query()` and `QueryRow()`, dosen't support `Exec()`. `Query()` and `QueryRow()` return result of any one shards.